frontend/src/lib/notification.tsx

```diff
@@ -55,6 +55,18 @@ export class Notification {
     // generate the id based on the message and the date attached to a notification
     this.id = btoa(unescape(encodeURIComponent(`${this.date},${this.message},${this.cluster}`)));
   }
+
+  set message(message: string) {
+    this._message = message;
+    if (this._message.length > 250) {
```

Comment:
I wonder if this message shortening is needed if we are already limiting it to 200 notifications.  It's in general good to apply limits, so probably good to keep it anyway. It's not clear to me how long the message could be or what the typical size is.

Reply:
I had a repeated notification which was less than 50 chars long. Very anecdotal but still all the notifications I had fit in well under 250 chars.
I think it's always okay to increase these limits in a controlled way, but I wanted to limit the chances of notifications hitting that limit, and the most variable size part of it is the message.